scripts/pre-push-gate.sh

```
@@ -19,6 +19,27 @@ cleanup() {
+# extension. Mkdocs.yml conflict in PR #1357 round 1 slipped through because
+# the local sweep filter only included *.go/*.json/*.templ/*.md. This check
+# runs in milliseconds and fail-fasts before the test suite eats 2-3 minutes.
+# Markers checked: <<<<<<< (start), ======= (separator), >>>>>>> (end), each
+# requiring a trailing space or EOL to avoid matching legitimate content like
+# a markdown ASCII rule of exactly seven equals.
+markers=$(git ls-files -z \
+    | xargs -0 grep -nE '^(<{7}|={7}|>{7})( |$)' 2>/dev/null \
+    | head -50 || true)
+if [ -n "$markers" ]; then
+    echo "FAIL: unresolved merge conflict markers in tracked files:"
+    echo "$markers" | sed 's/^/  /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)